lib/cloudinary/carrier_wave/storage.rb

```diff
@@ -89,7 +89,7 @@ def store_cloudinary_identifier(version, filename, resource_type=nil, type=nil)
         uploader.model.set(column, name)
       end
     elsif defined?(Neo4j::VERSION) && Neo4j::VERSION.split(".").first.to_i >= 5
-        uploader.model.set(column, name)
+        uploader.model.write_attribute(column, name)

```

Comment:
This is because for our project to work we need https://github.com/cloudinary/cloudinary_gem/pull/217